daemon/algod/api/algod.oas2.json

```diff
@@ -3193,6 +3262,25 @@
         "$ref": "#/definitions/Application"
       }
     },
+    "BoxesResponse": {
```

Comment:
So this means we return:
```
{ "boxes": ["name1", "name2", name3"]}
```
?
Do we avoid returning arrays directly, in case some readers have trouble with non-object responses?  I'd have expected:
```
["name1", "name2", "name3"...]
```
Though I could also imagine
```
[ {"name": "name1"}, {"name": "name2"}]
```
so that we could someday offer
```
[{"name": "name1", "value": "AAA"}...
```
if we'd like to.

Reply:
That makes sense, I am switching the implementation from
```
{ "boxes": ["name1", "name2", name3"]}
```
to the simplest
```
["name1", "name2", "name3"...]
```

We can always change later to
```
[{"name": "name1", "value": "AAA"}...]
```
if we decide to return contents of boxes, by changing array item model description.